ClientApp/viking-vault/src/display_user_list/AddMoneyModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import './AddMoney.css';
+
+interface IAddMoneyModalProps{
+    open: boolean;
+    userName: string;
+    closeModal: any;
+    addMoney: any;
+}
+
+interface IAddMoneyModalState{
+    amount: string;
+}
+
+class AddMoneyModal extends React.Component<IAddMoneyModalProps, IAddMoneyModalState>{ 
+    state = {
```

Comment:
You can write this in one line, if it's only 1 property is still readable, `state = { amount: "" }`